packages/wallet/test/services/TransactionsTracker.test.ts

```diff
@@ -867,5 +915,199 @@ describe('TransactionsTracker', () => {
       expect(inFlightTransactionsStore.set).toHaveBeenCalledTimes(3);
       expect(inFlightTransactionsStore.set).lastCalledWith([outgoingTx]);
     });
+
+    it('emit transaction from signed observable', async () => {
```

Comment:
It doesn't seem to be awaiting anything, same in other new tests
```suggestion
    it('emit transaction from signed observable', () => {
```

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1113/commits/cdbad467fb4a95811c6ca9c5c68bdb3c82b5a417